main/auth/external_login/facebook.inc.php

```diff
@@ -104,3 +127,20 @@ function change_to_valid_chamilo_login($in_txt) {
     return preg_replace("/[^a-zA-Z1-9_\-.]/", "_", $in_txt);
     exit;
 }
+
+/**
+ * Get user language
+ * @input : a string
+ *  @return : a string containing valid language
+ * */
+
+function get_language($language = 'pl_PL') {
```

Comment:
Default should be en_US, bracket should be on the next line. Spacing is bad for the following lines